app/experimenter/nimbus-ui/src/components/ApolloErrorAlert/index.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { ApolloError } from "@apollo/client";
+import React from "react";
+import { ApolloErrorAlert } from ".";
+import {
+  GQL_ERROR,
+  NETWORK_SERVER_ERROR,
+  NETWORK_SERVER_PARSE_ERROR,
+} from "./mocks";
+
+export default {
+  title: "components/ApolloErrorAlert",
+  component: ApolloErrorAlert,
+};
+
+export const withError = () => (
+  <ApolloErrorAlert
+    error={new Error("Uh oh. You made the app crashy crash.") as ApolloError}
+  />
+);
+
+export const withNetworkServerError = () => (
+  <ApolloErrorAlert error={NETWORK_SERVER_ERROR as unknown as ApolloError} />
```

Comment:
Q: Instead of casting `as unknown as ApolloError` in multiple files is it possible to define the data types in the ./mocks file when you're exporting the `const`s?

Reply:
Heh yeah good question, I'm copying what [Les did here](https://github.com/mozilla/experimenter/compare/issues/5765...lmorchard:issues/5765#diff-7eb756946081987e4b6fdab1479633ea511c0d72edcd4a69cb86ddb51e5b975dR30) which is basically what you suggested 😎